traffic-advice.md

```diff
@@ -0,0 +1,32 @@
+# Traffic Advice
+
+Publishers may wish not to accept traffic from [private prefetch proxies](README.md) and other sources other than direct user traffic, for instance to reduce server load due to speculative prefetch activity.
+
+We propose a well-known "traffic advice" resource, analogous to `/robots.txt` (for web crawlers), which allows an HTTP server to declare that implementing agents should stop sending traffic to it for some time.
+
+## Proposal
+
+Agents which respect traffic advice should fetch the well-known path `/.well-known/traffic-advice.json`. If it returns a `200 OK` response with the `application/json` MIME type, the response body should contain valid JSON like the following:
+
+```json
+[
+    {"user_agent": "prefetch-proxy", "disallow": true}
+]
+```
+
+Each agent has a series of identifiers it recognizes, in order of specificity:
+* its own agent name
+* decreasingly specific generic categories that describe it, like `"prefetch-proxy"`
+* `"*"` (which applies to every implementing agent)
+
+It finds the most specific element of the response, and applies the corresponding advice (currently only a boolean which advises disallowing all traffic) to its behavior. The agent should respect the cache-related response headers to minimize the frequency of such requests and to revalidate the resource when it is stale.
+
+Currently the only advice is the key `"disallow"`, which specifies a boolean which, if present and `true`, advises the agent not to establish connections to the origin. In the future other advice may be added.
+
+If the response has a `404 Not Found` status, on the other hand, the agent should apply its default behavior.
```

Comment:
Well, my general philosophy is the explainer doesn't need to be exhaustive, but should also avoid being inaccurate. So this sentence in particular is troubling because it's over-specific, and implies that e.g. 409s will not get the same treatment.

Reply:
I've added "or a similar status" to hopefully imply that I'm not saying 404 need be the only status with this treatment. I have difficult seeing why a 409 response would make sense here (can GET requests generally generate conflicts?), and some like 429 Too Many Requests might conceptually make sense to parse as "disallow" (if there are too many requests for what is basically a static resource, should we really be sending more non-essential traffic).

It definitely is the case that handling of the various 3XX, 4XX and 5XX statuses will need slightly more text than I want here. I only meant to say what happens in the two cases likely to happen in practice.